FirefoxLite_Downloader_Library/src/main/java/org/mozilla/firefoxlitedownloaderlibrary/Constants.java

```diff
@@ -0,0 +1,25 @@
+
+package org.mozilla.firefoxlitedownloaderlibrary;
+
+
+
+public final class Constants {
```

Comment:
where do you use this file?
Could you add a java-doc to explain what this file is for?